src/api/cvc4cppkind.h

```diff
@@ -134,15 +134,15 @@ enum CVC4_PUBLIC Kind : int32_t
    */
   LAMBDA,
   /**
-   * Hilbert choice (epsilon) expression.
+   * Witness expression.
```

Comment:
I think it'd be good to document what exactly is the "Witness" operator, i.e. how it differs from Hilbert choice.

Reply:
I don't understand the comment on the extra operand. And I think what would be good to document is that the "witness" operator or however we will call it is like Hilbert's choice except that it does not respect the axiom that forall x. F \equiv G -> choice x. F = choice x. G